plugins/lookup/onepassword.py

```diff
@@ -113,28 +120,32 @@
 class OnePassCLIBase(with_metaclass(abc.ABCMeta, object)):
     bin = "op"
 
-    def __init__(self, subdomain=None, domain="1password.com", username=None, secret_key=None, master_password=None):
+    def __init__(self, subdomain=None, domain="1password.com", username=None, secret_key=None, master_password=None,
+                 service_account_token=None):
         self.subdomain = subdomain
         self.domain = domain
         self.username = username
         self.master_password = master_password
         self.secret_key = secret_key
+        self.service_account_token = service_account_token
 
         self._path = None
         self._version = None
 
     def _check_required_params(self, required_params):
-        non_empty_attrs = dict((param, getattr(self, param, None)) for param in required_params if getattr(self, param, None))
-        missing = set(required_params).difference(non_empty_attrs)
-        if missing:
-            prefix = "Unable to sign in to 1Password. Missing required parameter"
-            plural = ""
-            suffix = ": {params}.".format(params=", ".join(missing))
-            if len(missing) > 1:
-                plural = "s"
-
-            msg = "{prefix}{plural}{suffix}".format(prefix=prefix, plural=plural, suffix=suffix)
-            raise AnsibleLookupError(msg)
+        if not self.service_account_token:
```

Comment:
This will skip parameter validation for `OnePassCLIv1`. Even if this won't be called by methods in a `OnePassCLIv1` object in the current plugin code, this should do the right thing when called in isolation.